.gitignore

```diff
@@ -1,4 +1,574 @@
 node_modules
-dist
 .DS_Store
 js/preload/_*.js
+
```

Comment:
this is a just a super generic list and many don't apply to this project.  to prevent pollution this should be stripped heavily stripped down.   e.g., why mongoSettings.xml, .pyc, etc.   i would prefer to keep minimal and add as necessary.